optuna/progress_bar.py

```diff
@@ -75,15 +82,21 @@ def _init_valid(self) -> None:
         optuna_logging.disable_default_handler()
         optuna_logging._get_library_root_logger().addHandler(_tqdm_handler)
 
-    def update(self, elapsed_seconds: float) -> None:
+    def update(self, elapsed_seconds: float, study: "Study") -> None:
         """Update the progress bars if ``is_valid`` is :obj:`True`.
 
         Args:
             elapsed_seconds:
                 The time past since :func:`~optuna.study.Study.optimize` started.
+            study:
+                The current study object.
         """
 
         if self._is_valid:
+            if not study._is_multi_objective():
```

Comment:
I realised that when a study has only non-complete trials such as a failed trial, the current code does not work as we expected. 
```suggestion
            if not study._is_multi_objective() and study.get_trials(states=[TrialState.COMPLETE]):
```

Note that we add the following import statement about L10.

```python
from optuna.trial import TrialState
```